cmd/_integration-tests/transport/handlers/server/server_handler.go

```diff
@@ -63,3 +64,8 @@ func (s transportService) CtxToCtx(ctx context.Context, in *pb.MetaRequest) (*pb
 
 	return &resp, nil
 }
+
+// ErrorRPC implements Service.
+func (s transportService) ErrorRPC(ctx context.Context, in *pb.Empty) (*pb.Empty, error) {
+	return nil, fmt.Errorf("This error should be json over http transport")
```

Comment:
This is a test, so, not a big deal, and this is probably known, but as a reminder: under normal circumstances, if an error is static, declare it with `errors.New` at the package level and just reference it here.